Source/WebInspectorUI/UserInterface/Views/DataGrid.css

```diff
@@ -365,6 +365,7 @@ body[dir=rtl] .data-grid .resizer {
     background: none;
     border: none;
     outline: none;
+    text-overflow: ellipsis;
```

Comment:
do we really want to do this for _all_ `WI.DataGrid`?  could/should we just do it in the Node panel for now?

Reply:
I think we should have this affordance for all `WI.DataGrid` to mitigate the confusion from clipped values which at first glance look wrong/incomplete.